### PR TITLE
Replace models 103 and 107 with 248 and 249

### PR DESCRIPTION
**Pull request by sovrasov**

### CODE REVIEW

File: models/intel/person-reidentification-retail-0248/description/person-reidentification-retail-0248.md [hunk 1]
```diff
@@ -10,14 +10,14 @@ feature map outputs an embedding vector of 256 floats.
...
-![](./person-reidentification-retail-0107.jpg)
+![](./person-reidentification-retail-0248.jpg)
 
 ## Specification
 
 | Metric                            | Value                                     |
 |-----------------------------------|-------------------------------------------|
-| Internal-reID-test rank@1 accuracy| 91.7%                                     |
-| Internal-reID-test mAP            | 63.4%                                     |
+| Market-1501 rank@1 accuracy       |  %                                    |
```
Comment (vladimir-dudnik) [inline models/intel/person-reidentification-retail-0248/description/person-reidentification-retail-0248.md]: is expected accuracy known for the model?

File: models/intel/person-reidentification-retail-0249/description/person-reidentification-retail-0249.md [hunk 2]
```diff
@@ -10,14 +10,14 @@ feature map outputs an embedding vector of 256 floats.
...
-![](./person-reidentification-retail-0103.jpg)
+![](./person-reidentification-retail-0249.jpg)
 
 ## Specification
 
 | Metric                            | Value                                     |
 |-----------------------------------|-------------------------------------------|
-| Internal-reID-test rank@1 accuracy| 93.5%                                     |
-| Internal-reID-test mAP            | 69.5%                                     |
+| Market-1501 rank@1 accuracy       |  %                                    |
```
Comment (vladimir-dudnik) [inline models/intel/person-reidentification-retail-0249/description/person-reidentification-retail-0249.md]: accuracy metric missed?

File: models/intel/person-reidentification-retail-0248/description/person-reidentification-retail-0248.md (hunk 1 above)
Comment (sovrasov) [reply models/intel/person-reidentification-retail-0248/description/person-reidentification-retail-0248.md]: The accuracy is known

File: models/intel/person-reidentification-retail-0249/description/person-reidentification-retail-0249.md (hunk 2 above)
Comment (sovrasov) [reply models/intel/person-reidentification-retail-0249/description/person-reidentification-retail-0249.md]: Fixed

File: tools/accuracy_checker/configs/person-reidentification-retail-0249.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          - FP32-INT8
+        model:   intel/person-reidentification-retail-0249/FP32-INT8/person-reidentification-retail-0249.xml
+        weights: intel/person-reidentification-retail-0249/FP32-INT8/person-reidentification-retail-0249.bin
+        adapter: reid
+
+      - framework: dlsdk
+        tags:
+          - FP16-INT8
+        model:   intel/person-reidentification-retail-0300/FP16-INT8/person-reidentification-retail-0249.xml
+        weights: intel/person-reidentification-retail-0300/FP16-INT8/person-reidentification-retail-0249.bin
```
Comment (eaidova) [inline tools/accuracy_checker/configs/person-reidentification-retail-0249.yml]:
```suggestion
        model:   intel/person-reidentification-retail-0249/FP16-INT8/person-reidentification-retail-0249.xml
        weights: intel/person-reidentification-retail-0249/FP16-INT8/person-reidentification-retail-0249.bin
```

File: tools/accuracy_checker/configs/person-reidentification-retail-0248.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+          - FP32-INT8
+        model:   intel/person-reidentification-retail-0248/FP32-INT8/person-reidentification-retail-0248.xml
+        weights: intel/person-reidentification-retail-0248/FP32-INT8/person-reidentification-retail-0248.bin
+        adapter: reid
+
+      - framework: dlsdk
+        tags:
+          - FP16-INT8
+        model:   intel/person-reidentification-retail-0300/FP16-INT8/person-reidentification-retail-0248.xml
+        weights: intel/person-reidentification-retail-0300/FP16-INT8/person-reidentification-retail-0248.bin
```
Comment (eaidova) [inline tools/accuracy_checker/configs/person-reidentification-retail-0248.yml]:
```suggestion
        model:   intel/person-reidentification-retail-0249/FP16-INT8/person-reidentification-retail-0248.xml
        weights: intel/person-reidentification-retail-0249/FP16-INT8/person-reidentification-retail-0248.bin
```

File: tools/accuracy_checker/configs/person-reidentification-retail-0249.yml (hunk 3 above)
Comment (sovrasov) [reply tools/accuracy_checker/configs/person-reidentification-retail-0249.yml]: fixed

File: tools/accuracy_checker/configs/person-reidentification-retail-0248.yml (hunk 4 above)
Comment (sovrasov) [reply tools/accuracy_checker/configs/person-reidentification-retail-0248.yml]: fixed

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Pull request merged**